client/me/purchases/remove-purchase/style.scss

```diff
@@ -19,8 +19,3 @@
 		text-overflow: ellipsis;
 	}
 }
```

Comment:
While we're here, I notice that the lines above this aren't used at all. There doesn't appear to be an h1 in the dialog as far as I can tell, so we should get rid of that too (unless I'm missing something).
```css
 .remove-purchase__dialog.dialog__content {
 	h1 {
 		height: auto;
 		overflow: hidden;
  		text-overflow: ellipsis;
  	}
  }
```